scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -1011,7 +1480,8 @@ def start( self ):
             if self.__always_use_docker or (self._container_runtime == 'docker' and not self.__always_use_cri):
                 global_log.info('kubernetes_monitor is using docker for listing containers')
                 self.__client = DockerClient( base_url=('unix:/%s'%self.__socket_file), version=self.__docker_api_version )
-                self._container_enumerator = DockerEnumerator( self.__client, self.__agent_pod )
+                self._container_enumerator = DockerEnumerator( self.__client, self.__agent_pod,
```

Comment:
Presumably this is only for Docker because that's all we're testing for at the moment and in the future this will also be added to other CRI runtimes?